src/sentry/static/sentry/app/views/performance/trends/changedProjects.tsx

```diff
@@ -0,0 +1,306 @@
+                            {getDescription(trendChangeType, trendView, projectTrend)}
+                          </div>
+                        </ProjectTrendContainer>
+                      </React.Fragment>
+                    ) : (
+                      <ProjectTrendContainer>
+                        <div>{getNoResultsDescription(trendChangeType)}</div>
+                      </ProjectTrendContainer>
+                    )}
+                    {projectTrend && (
+                      <ButtonContainer>
+                        <Button
+                          onClick={() =>
+                            handleViewTransactions(projectTrend, projects, location)
+                          }
+                          size="small"
+                        >
+                          {t('View Transactions')}
+                        </Button>
+                      </ButtonContainer>
+                    )}
+                  </React.Fragment>
+                )}
+              </DescriptionContainer>
+              <VisualizationContainer>
+                {projectTrend &&
+                  !isLoading &&
+                  getVisualization(trendChangeType, projectTrend)}
+              </VisualizationContainer>
+            </StyledPanel>
+          </ChangedProjectsContainer>
+        );
+      }}
+    </DiscoverQuery>
+  );
+}
+
+const StyledPanel = styled(Panel)`
+  display: flex;
+  flex-direction: row;
+`;
+const DescriptionContainer = styled('div')`
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  min-height: 185px;
+`;
+const VisualizationContainer = styled('div')``;
+const ChangedProjectsContainer = styled('div')``;
+const ContainerTitle = styled('div')`
+  padding-top: ${space(3)};
+  padding-left: ${space(2)};
+`;
+const LoadingIndicatorContainer = styled('div')`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+`;
+const ProjectTrendContainer = styled('div')`
+  padding: ${space(2)};
+
+  margin-top: ${space(1)};
+  margin-left: ${space(1)};
+
+  font-size: 14px;
+  color: ${p => p.theme.gray600};
+`;
+const ButtonContainer = styled('div')`
+  padding-left: ${space(2)};
+  margin-left: ${space(1)};
+  padding-bottom: ${space(2)};
+`;
+
+function getVisualization(
+  trendChangeType: TrendChangeType,
+  projectTrend: NormalizedProjectTrend
+) {
+  const color = trendToColor[trendChangeType];
+
+  const trendPercent = formatPercentage(
+    projectTrend.percentage_aggregate_range_2_aggregate_range_1 - 1,
+    0
+  );
+
+  return (
+    <div>
+      <TrendCircle color={color}>
+        <TrendCircleContent>
+          <TrendCirclePrimary>
+            {trendChangeType === TrendChangeType.REGRESSION ? '+' : ''}
+            {trendPercent}
+          </TrendCirclePrimary>
+          <TrendCircleSecondary>{projectTrend.project}</TrendCircleSecondary>
+        </TrendCircleContent>
+      </TrendCircle>
+    </div>
+  );
+}
+
+const TrendCircle = styled('div')<{color: string}>`
```

Comment:
Should we reuse the circleIndicator component here?

Reply:
Not sure about that, I need a circular border with text inside, circle indicator seems to just currently be a circle with a background colour, I don't feel the intent is similar enough to merge them.